### PR TITLE
New Article: Cloud Backup Troubleshooting

### PR DESCRIPTION
**Pull request by BadCrumpany**
New Article: Cloud Backup Troubleshooting on Windows 2012 Servers

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-bd03a44ad0/how-to/)

**Comment by chri2547** [comment]
@BadCrumpany, I am working on getting this article published for you. I notice there is another article that is very similar to the one you wrote: https://support.rackspace.com/how-to/cloud-backup-troubleshooting/

Do you think we ought to combine these articles? Or do you feel as though what you have provided should stand on its own?

Thanks!
Chris

**Comment by BadCrumpany** [comment]
Hi Chris,
I looked over the one you specified and it looks like that one focuses on people that are already semi-technical.  The one I was writing was to walk people with little experience through the initial troubleshooting steps.  It was something that people in chat were getting asked about so it was created to remove some of that work from the chatters.  If you think it would be more beneficial to merge these steps in that one, I understand but it looks like they are focused at two different tech tiers of customers (technical vs non-technical).

Thanks for the help!

**Comment by BadCrumpany** [comment]
Also, these steps are specific to Windows 2012 Cloud Servers.

**Comment by chri2547** [comment]
Ah, I see what you mean. Then perhaps we change the title a bit to reflect that this is basic troubleshooting on Windows. Something like "Cloud Backup - Basic troubleshooting on Windows 2012 Servers"

**Comment by BadCrumpany** [comment]
Sounds like a winner!

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-50226c2442/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-348a4b4273/how-to/)

**Comment by cyrichardson** [comment]
@chri2547 Suggested changes here: 
[cloud-backup-troubleshooting-on-windows-2012-servers_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2287208/cloud-backup-troubleshooting-on-windows-2012-servers_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-823ccfa827/how-to/). :bow:

**Comment by chri2547** [comment]
@BadCrumpany, in this article you want folks to be able to link to the Download Backup Log From Control Panel article. Is that the second article you wrote and that Kate worked on? Or is it something else?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b3cbd3ae65/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1145a8c624/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0efc0827e0/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-442396ef76/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d6da2861ef/how-to/). :bow:

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+If there are no open support tickets about the server in question, in the next section, attempt to establish a connection.
+
+## Establish connection 
+
+1. In the [Rackspace Cloud Control Panel](https://mycloud.rackspace.com), click on **Backups > Systems**.
+2. From the list of Cloud Backup Systems, click the backup server name. 
+   This action displays the following details about the backup system:
+   
+   - the IP address of the server
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: ... The Internet Protocol (IP) address ...

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+   - the system type
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: ... The ...

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+   - the backup vault size
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: ... The ...

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+   - if encryption is enabled
+   - the server region
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: ... The ...

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+   - the backup region
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: ... The ...

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+   - if service net is enabled
+   - the agent version
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: ... The ...

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+2. From the list of Cloud Backup Systems, click the backup server name. 
+   This action displays the following details about the backup system:
+   
+   - the IP address of the server
+   - the system type
+   - the backup vault size
+   - if encryption is enabled
+   - the server region
+   - the backup region
+   - if service net is enabled
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: ... If ...

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+## Establish connection 
+
+1. In the [Rackspace Cloud Control Panel](https://mycloud.rackspace.com), click on **Backups > Systems**.
+2. From the list of Cloud Backup Systems, click the backup server name. 
+   This action displays the following details about the backup system:
+   
+   - the IP address of the server
+   - the system type
+   - the backup vault size
+   - if encryption is enabled
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: ... If ...

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+4. Wait ten seconds to ensure that the status is accurate because it can change after a few seconds.
+5. After 10 seconds, perform one of the following actions:
+ - If the status is **connected**, contact Rackspace Technical Support.
+ - If the status is **disconnected**, in the next section, restart the DriveClient service.
+
+## Restart the DriveClient service
+
+1. Log into the server via RDP and click the **Windows** symbol in the bottom-left corner.
+2. Click the **Search** icon in the top-right corner.
+3. Enter **services.msc** in the search field, and then click **services** that populates under the search bar.
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: [is services a word that populates under the search bar? If so, is it capitalized?]

File: content/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+   
+   b. Click the drop-down list next to **Startup type** and select **Automatic** (if not already selected).
+  
+   c. Click the **OK** button at the bottom of the page.
+   
+   d. After the service restarts, check the Control Panel to ensure the service is running.
+   
+7. Perform one of the following:
+   - If the agent is connected, contact Rackspace Technical Support.
+   - If the agent is not connected, attach the backup log to a technical support ticket. The backup log is located at C:\ProgramData\Driveclient\log\driveclient.log. Rackspace Support who will use the log file to expedite a response to the issue.
```
Comment (catlook) [inline content/cloud-backup-troubleshooting-on-windows-2012-servers.md]: ...  at **C:\ProgramData\Driveclient\log\driveclient.log**. Rackspace Support will use ...

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-e2cd44c734/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a8d85c2612/how-to/).

**Pull request merged**